src/drag-and-drop/components/container.tsx

```diff
@@ -226,7 +230,9 @@ export const Container: React.FC<IProps> = ({ authoredState, interactiveState, s
   const draggingAreaStyle = {
     width: canvasWidth + "px",
     height: canvasHeight + "px",
-    backgroundImage: authoredState.backgroundImageUrl ? cssUrlValue(authoredState.backgroundImageUrl) : undefined
+    backgroundImage: authoredState.backgroundImageUrl ? cssUrlValue(authoredState.backgroundImageUrl) : undefined,
+    transform: report && view !== "standalone" ? `scale(${scaleRatioForReport})` : undefined,
```

Comment:
I'm guessing you tested this, but often when applying scale changes in css mouse pointer and touch events can get messed up. You'd usually see the issue when trying to click and drag something. 
This issue is why we usually do scaling at the iframe level instead of inside of the interactive.  But in this case this approach seems to make sense, as long as we test it and fix any pointer event issues.

Reply:
@scytacki In the context of a teacher report/dashboard, do we want to allow the interactive to be manipulated? When working on this it seemed like the interactive was specifically frozen to not allow it to be changed, but maybe I'm wrong about that.